src/core/DY.Core.Bytes.fst

```diff
@@ -1270,6 +1342,7 @@ let get_label_pk_enc #cusages pk nonce msg =
 
 /// User lemma (public-key decryption bytes invariant).
 
+#push-options "--z3rlimit 25"
```

Comment:
Do we need to change the rlimit here? This seems unrelated to the KEM implementation.

Reply:
Yes we need, I don't really know why, but sometimes the rlimit must be bumped when the SMT gods decide so. Here, it might be because `bytes` has more constructors.